src/workshops/applicant-git-2/index.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Git introduction
+description: This workshop is an introduction to using Git for version control; GitHub for hosting a codebase and deploying a website; and the terminal for creating files and moving between directories.
+tags:
+  - workshop
+  - git
+keywords:
+  - git
+  - introduction
+starter: false
+---
+
+This workshop will help you practise with Git on the terminal. Follow these steps to connect a local repository to a remote one on GitHub. Before following these steps, you'll need to [install Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git).
+
+This workshop is an advancement on concepts covered in [our Git introduction](../applicant-git-1/). Completing that workshop is not a requirement to understand this one, however you might prefer to focus on the fundamentals there before diving into this workshop.
+
+This workshop assumes you have a working knowledge of [using the terminal](https://www.joshwcomeau.com/javascript/terminal-for-js-devs/).
+
+## Getting started
+
+Create a folder on your computer and add files for HTML, CSS and JavaScript. Add some content to the HTML, link the CSS and add a script tag which points to your JS file.
```

Comment:
Not a huge deal but it might be good to encourage best practices by having them init the Git repo before writing any code. Otherwise it makes it seem as though tracking changes is something they can tack on later/at the end

Reply:
Oh yeah, good shout - the order isn't quite right here